extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -106,15 +107,19 @@ async function handleObjectPropertiesDialogCommand(context: azdata.ObjectExplore
 		return;
 	}
 	try {
-		const parentUrn = await getParentUrn(context);
+		const parentUrn = context.nodeInfo ? await getParentUrn(context) : undefined;
+		const objectType = context.nodeInfo ? context.nodeInfo.nodeType as ObjectManagement.NodeType : ObjectManagement.NodeType.Server;
```

Comment:
Databases can be top-level nodes too. Have you talked with @erinstellato-ms about what we want to do in that case?

Reply:
I wasn't aware of this case. I'll sync with Erin

Reply:
When connected directly to database, then the properties will open the Database Properties dialog.